### ISSUE TITLE
Unable to sign in using OAuth

### ISSUE THREAD

**Issue by devcamcar** [julian-alarcon/prospect-mail#156]
When I enter my work email address and click Next, I then see"Taking you to your organization's sign in page ...".  But after a few seconds it just takes me back to the initial sign in page.

Is this something that is supported currently?

Thanks!

**Comment by julian-alarcon** [comment]
Can you try the latest pre-release? https://github.com/julian-alarcon/prospect-mail/releases/tag/v0.3.0

**Comment by gomker** [comment]
Getting the same issue although I am having a problem using RSA auth, it times out before I can use my key or enter my token. I have just tried v0.3 - it waits a bit longer but still gives a timeout.

**Comment by julian-alarcon** [comment]
We have settings.json file, can you test adding your custom oauth domains to `urlsExternal` something like

{
 "urlsExternal":["outlook.live.com", "outlook.office365.com", "outlook.office.com", "your-oath-domain.com"]
}

https://github.com/julian-alarcon/prospect-mail#settings 


For your case @Colmanetti could be : `adfs.bb.com.br` and `interno.bb.com.br`

**Comment by daveola** [comment]
Thanks for pointing me to the AppImage - that fix seemed to work (without any settings changes)

**Comment by pgalvarez-elsevier** [comment]
Hello, I am facing this issue too.
I tried adding the ADFS/SSO endpoint of my company in the settings.json file to no avail.

Here's the log when running `prospect-mail --enable-logging`:
```
[14239:0128/164621.847608:INFO:CONSOLE(4)] "Uncaught ReferenceError: require is not defined", source: https://my.company.com/adfs/ls/wia?client-request-id=VER_LONG_CLIENT_REQUEST_REDACTED (4)
```

I redacted the client-request-id from the URI, but I can confirm that If I click on that link and open it on a firefox instance, I get prompted for MFA and then access https://outlook.office.com/mail/ (after a brief jump via https://login.microsoftonline.com/login.srf)

Any hint?

**Comment by dekiertanna** [comment]
The issue still persists on build 0.4.0. My organization is using OAuth also, and after providing an email, instead of being prompted to authenticate, the application just shows a white screen. 
[9353:0525/100714.252697:INFO:CONSOLE(4)] "Uncaught ReferenceError: require is not defined"
Of course I will not show the rest if the message, for security reason.

**Comment by xJason6** [comment]
I encountered the same issue (version 0.4.0).
Workaround with settings.json editing doesn't work.
Strangely teams-for-linux app, which is also using Electron, handles that quite well...

**Comment by raul72** [comment]
> "Uncaught ReferenceError: require is not defined", source: ...

For me, as additional note, our organization auth domain has basic http-auth before it

Prospect mail shows white page and logs the "require is not defined" error instead of prompting for authentication

When opening the source URL in browser, auth is prompted

**Comment by raul72** [comment]
I was using Appimage not Snap - but yes, when I remove `~/.config/Prospect Mail/` 
- `Prospect-Mail-0.4.0.appimage`- white page after "Taking you to your organization's sign-in page"
- `Prospect-Mail-0.5.0-beta1.appimage` works without requiring tinkering

**Comment by julian-alarcon** [comment]
So, can you confirm that its fixed @raul72 ?

**Comment by raul72** [comment]
For me, the problem is fixed, yes

**Comment by julian-alarcon** [comment]
Thanks @raul72 !
Fixed by #288

**Comment by jdegiac** [comment]
App functioning, but when I try and switch accounts to my gmail account it shows a blank white screen
